### PR TITLE
feat(ui): display CSV columns with sample values

### LINKED ISSUE #14: Mapper: display CSV file structure
The mapper needs to show the structure of a CSV file.

It shows:
- The name of the CSV file
- The name of each column
- For each column, a small sample of the values it contains

Screenshot 2020-09-15 at 16 16 29

Depends on #62

### PR DESCRIPTION
**Pull request by martinmaillard**
Will close #14

### CODE REVIEW

File: ui/src/api/index.ts [hunk 1]
```diff
@@ -25,6 +26,7 @@ Hydra.resources.factory.addMixin(ProjectMixin)
 Hydra.resources.factory.addMixin(CSVMappingMixin)
 Hydra.resources.factory.addMixin(CSVSourceCollectionMixin)
 Hydra.resources.factory.addMixin(CSVSourceMixin)
+Hydra.resources.factory.addMixin(CSVColumnMixin)
```
Comment (tpluscode) [inline ui/src/api/index.ts:29]: As this keeps growing, you might combine all mixins in a single module and add them all at once

```js
import * as Mixins from './mixins'

Hydra.resources.factory.addMixin(...Object.values(Mixins))
```

**Review by tpluscode — APPROVED** [review]

File: ui/src/api/index.ts (hunk 1 above)
Comment (martinmaillard) [reply ui/src/api/index.ts:29]: Doesn't that mean I still have to list all the mixins in `mixins/index.ts`?
And TypeScript is not too happy about this, as it doesn't know the type of the values of `Mixins`.

Comment (tpluscode) [reply ui/src/api/index.ts:29]: Yes, would still have to `export` from `index.ts`.

It's just a suggestion. Feel free to ignore :)

Comment (martinmaillard) [reply ui/src/api/index.ts:29]: I'll keep it here for now, but will probably extract at some point

**Pull request merged**